[rules_rust] Break a circular dependency

https://github.com/bazelbuild/rules_rust/issues/2383 claims that this
is already fixed, but clearly it is not.

--- util/import/BUILD.bazel
+++ util/import/BUILD.bazel
@@ -20,10 +20,7 @@
 
 alias(
     name = "import",
-    actual = select({
-        ":use_fake_import_macro": ":fake_import_macro_impl",
-        "//conditions:default": ":import_macro_label",
-    }),
+    actual = ":fake_import_macro_impl",
     visibility = ["//visibility:public"],
 )
 
